x/distribution/spec/01_concepts.md

```diff
@@ -3,12 +3,21 @@ order: 1
 -->
 
 # Concepts
+In Proof of Stake (PoS) blockchains, rewards gained from transaction fees are paid to validators. The fee distribution module fairly distributes the rewards to the validators' constituent delegators. 
+
+Rewards are calculated per period. The period is updated each time a validator's delegation changes, for example, when the validator receives a new delegation.
```

Comment:
I couldn't find the definition of a period anywhere so I think it'd be helpful to define what a period is in the Cosmos context.

Reply:
yes! good observation @shahankhatch tag me when we get a definition

Reply:
That would refer to the F1 concept. Not sure how much we should rewrite the specification of F1 here. If we want to explain the meaning of periods, we need to explain F1 in basically the same amount of detail as the relevant doc.